sdk/src/main/scala/com/horizen/account/mempool/MempoolMap.scala

```diff
@@ -0,0 +1,149 @@
+package com.horizen.account.mempool
+
+import com.horizen.SidechainTypes
+import com.horizen.account.transaction.EthereumTransaction
+import scorex.util.{ModifierId, ScorexLogging}
+
+import java.math.BigInteger
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+import scala.util.Try
+
+case class MempoolMap() extends ScorexLogging {
```

Comment:
`case` can be removed.
check the IDEA suggestions to this class.

Reply:
I was about to the comment same further down: There are a lot of useful suggestions in IDEA in this class.
Can I also recommend a reformat with `scalafmt` as this file is totally new? *duck and cover*

Reply:
Done. I have configured IntelliJ to use scalafmt and reformatted the file.